index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
+		<img src="assets/more-pills.svg" alt="a blue pill, a purple pill, and a yellow pill">
+		
+		<span class="subTitle">A Viget Exploration</span>
+		<h1>
+			<img src="assets/title.svg" alt="Improving Medical Adherence">	
+		</h1>
+		
+		<img src="assets/pills.svg" alt="a blue pill and yellow pill">
+		<img src="assets/purple-pills.svg" alt="two purple pills">
+
+		<!-- yellow-pill.svg is top left -->
+		<!-- more-pills.svg is top right -->
+		<!-- pills.svg is bottom left -->
+		<!-- purple-pills.svg is bottom right -->
+
+		<section class="whiteBox">
```

Comment:
BEM is just a system for creating classes.

A chunk of code will get a kabob-based class: `white-box`
 
Then, something nested in that gets a class like this: `white-box__heading`
Note that that is two "_" (Github doesn't render it well). That symbol notes nesting, so you have some sense of hierarchy just by looking at the class name. And you can keep going:
`white-box__heading__small-heading`

Then there's the "--" characters (again that's two "-") which denotes an alternate version, like so:
`white-box--large-padding`

You can name things however you like, it's just the symbols that are an important addition.  They just give more context to your class names, given the system.